packages/cli/src/server.ts

```diff
@@ -288,6 +289,16 @@ export class Server extends AbstractServer {
 		await eventBus.initialize();
 		Container.get(LogStreamingEventRelay).init();
 
+		// ----------------------------------------
+		// Workflow Indexing Setup
+		// ----------------------------------------
+		if (Container.get(WorkflowsConfig).indexingEnabled) {
+			const { WorkflowIndexService } = await import(
+				'@/modules/workflow-index/workflow-index.service'
+			);
+			Container.get(WorkflowIndexService).init();
+		}
```

Comment:
Could we move this to a separate method, eg `initWorkflowIndexing` to we don't increase the complexity of this and won't need the eslint ignore?